cluster/images/conformance/conformance-e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kubectl create -f conformance-e2e.yaml
+while true; do
+  STATUS=$(kubectl -n conformance get pods e2e-conformance-test -o jsonpath="{.status.phase}")
+  timestamp=$(date +"[%H:%M:%S]")
+  echo "$timestamp Pod status is: ${STATUS}"
+  if [[ "$STATUS" == "Succeeded" ]]; then
+    echo "$timestamp Done."
+    break
+  else
+    sleep 5
+  fi
+done
+NODE_NAME=$(kubectl get po -n conformance e2e-conformance-test -o 'jsonpath={.spec.nodeName}')
+docker exec ${NODE_NAME} tar cvf - /tmp/results | tar --strip-components 2 -xf -
```

Comment:
can we use `kubectl exec` instead?

Reply:
@ixdy not in this specific case. the files are in kind-worker1 or kind-worker2 containers (not in the pods as the pods go away by this time). 

So we need a new `kind export` for files (in addition to the existing logs) if we want to be agnostic

Reply:
the name of this script is misleading then, since this wouldn't necessarily work on a real cluster, right?

Reply:
oh, I see you removed this.

Reply:
yep, removed it